dot/network/service.go

```diff
@@ -192,8 +192,8 @@ func (s *Service) updateNetworkState() {
 func (s *Service) receiveCoreMessages() {
 	for {
 		// receive message from core service
-		msg := <-s.msgRec
-		if msg == nil {
+		msg, ok := <-s.msgRec
```

Comment:
same here

Reply:
msg needs to be defined in a separate line so it can be used at line 202 and onwards